### DISCUSSION TITLE
[EN] Gestion fails to add new part to quote #260

### DISCUSSION THREAD

**Discussion by LazyCoderOZ** [baimard/gestion#261] [Maybe a bug ?] [answered]
I am slightly unsure how you want me to provide extra information to this issue, I have a work around I will put in discussions as to keep it away from the issues list.

The database table oc_gestion_produit_devis has the column comment which does not have a default value.
I suspect the first line 
```
General error: 1364 Field 'comment' doesn't have a default value
```
is due to the database not liking a column being referenced without a default value.
Is it possible to set a default value for this column, or maybe get the database to allow null values?

My current work around is to delete the column comment in table oc_gestion_produit_devis and then roll back to gestion version 2.0.9
My code for doing this is below
```
$> mysql -uroot -p<password>
mysql []> use database <database>;
```
Check to make sure the column comment is in existence
```
mysql [<database>] SELECT * FROM oc_gestion_produit_devis;
```
Now drop the column
```
mysql [<database>] ALTER TABLE oc_gestion_produit_devis DROP COLUMN comment;
```
Then check the column is now gone
```
mysql [<database>] SELECT * FROM oc_gestion_produit_devis;
```
I shall continue to investigate to see if I can find a better more appropriate cure for the issue.

**Comment by baimard** [comment, marked as answer]
solved
